Sources/muterCore/CLICommands/RunCommand/Steps/DiscoverSourceFiles.swift

```diff
@@ -1,20 +1,28 @@
 import Foundation
+import Pathos
+import Curry
 
 struct DiscoverSourceFiles: RunCommandStep {
     private let notificationCenter: NotificationCenter = .default
     
     func run(with state: AnyRunCommandState) -> Result<[RunCommandState.Change], MuterError> {
-        
         notificationCenter.post(name: .sourceFileDiscoveryStarted, object: nil)
-        
-        let sourceFileCandidates = discoverSourceFiles(inDirectoryAt: state.tempDirectoryURL.path,
-                                                       excludingPathsIn: state.muterConfiguration.excludeList)
-        
+
+        let tempDirectoryURL = state.tempDirectoryURL
```

Comment:
if you're going to create this constant you should use it in more than one place, otherwise i would get rid of it

Reply:
oh, that should not be there, good catch...deleting...!